### DISCUSSION TITLE
【功能建议】港澳台限制解锁

### DISCUSSION THREAD

**Discussion by Bang0518** [the1812/Bilibili-Evolved#1609] [功能建议 / Ideas]
rt，建议增加此类功能，翻蔷实在有点不方便

**Comment by Coulomb-G** [comment]
不加

**Comment by Bang0518** [comment]
卧槽，这回复也太快了




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "the1812/Bilibili-Evolved"                                                                                    <notifications@github.com&gt;;
发送时间:&nbsp;2021年3月1日(星期一) 下午3:45
收件人:&nbsp;"the1812/Bilibili-Evolved"<Bilibili-Evolved@noreply.github.com&gt;;
抄送:&nbsp;"RiME"<1075733128@qq.com&gt;;"Author"<author@noreply.github.com&gt;;
主题:&nbsp;Re: [the1812/Bilibili-Evolved] 【功能建议】港澳台限制解锁 (#1609)





 
不加
